com.unity.multiplayer.mlapi/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This is the initial experimental Unity MLAPI Package, v0.1.0.
 - Refactored a new standard for Remote Procedure Call (RPC) in MLAPI which provides increased performance, significantly reduced boilerplate code, and extensibility for future-proofed code. MLAPI RPC includes `ServerRpc` and `ClientRpc` to execute logic on the server and client-side.
 - Added standarized serialization types, including built-in and custom serialization flows. See [RFC #2](https://github.com/Unity-Technologies/com.unity.multiplayer.rfcs/blob/master/text/0002-serializable-types.md) for details.
 - `INetworkSerializable` interface replaces `IBitWritable`.
+- Added `NetworkSerializer`... and `BitSerializer`..., which is the main aggregator that implements serialization code for built-in supported types and holds `BitReader` and `BitWriter` instances internally.
```

Comment:
nope, `NetworkSerializer` is the `BitSerializer` with a different name. we initially named it `BitSerializer` but changed it to `NetworkSerializer`. so basically, we're only adding `NetworkSerializer` :) (I know it was confusing, sorry!)

Reply:
All good! editing!

Reply:
also please replace `BitReader` and `BitWriter` with `NetworkReader` and `NetworkWriter`.